packages/provider/src/schedulers/captchaScheduler.ts

```diff
@@ -36,22 +33,28 @@ export async function storeCaptchasExternally(
 			: defaultSchedule
 		: defaultSchedule;
 
-	const job = new CronJob(cronSchedule, async () => {
-		const taskRunning = await checkIfTaskIsRunning(
-			ScheduledTaskNames.StoreCommitmentsExternal,
-			env.getDb(),
-		);
-		env.logger.info(
-			`${ScheduledTaskNames.StoreCommitmentsExternal} task running: ${taskRunning}`,
+	const job = new CronJob(cronSchedule, () => {
+		console.log(`Creating worker - from main thread: ${threadId}`);
+		const worker = new Worker(
+			path.resolve("../provider/dist/workers/storeCaptchaWorker.js"),
+			{
+				workerData: { config },
+			},
 		);
-		if (!taskRunning) {
-			env.logger.info(
-				`${ScheduledTaskNames.StoreCommitmentsExternal} task....`,
-			);
-			await tasks.clientTaskManager.storeCommitmentsExternal().catch((err) => {
-				env.logger.error(err);
-			});
-		}
+
+		worker.on("message", (message) => {
+			console.log(`Worker message: ${message}`);
```

Comment:
Please use the logger for consistency. We'll be changing all of this to JSON output using the @prosopo/common logger soon.